clearlinux/README-short.txt

```diff
@@ -0,0 +1 @@
+Official docker build of Clear Linux OS for Intel Architecture
```

Comment:
(for context, this is the blurb that'll show up in `docker search` and at the top of the Hub description page)